designer/server/src/main/scala/pl/touk/nussknacker/ui/api/description/MigrationApiEndpoints.scala

```diff
@@ -98,6 +115,88 @@ class MigrationApiEndpoints(auth: EndpointInput[AuthCredentials]) extends BaseEn
 object MigrationApiEndpoints {
 
   object Dtos {
+    sealed trait MigrationError
+
+    object MigrationError {
```

Comment:
I'd propose more human names for the errors:
* `MigrationError.InvalidScenario`
* `MigrationError.CannotMigrateArchivedScenario`
* `MigrationError.InsufficientPermission(missingPermission: Permission)`